.github/workflows/test.yml

```diff
@@ -86,3 +86,16 @@ jobs:
       with:
         name: Selenium HTML
         path: "${{ env.TEST_RESULTS_PATH }}/*.html"
+
```

Comment:
This may be set up for non-community edition of SonarQube 🤷 ?

See quite different instructions that are project-specific on https://sonarcloud.io/project/configuration/GitHubActions?id=eebbesen_StpFoodBlazor